distributed/dashboard/components/scheduler.py

```diff
@@ -2164,10 +2164,19 @@ def update(self):
 
 
 class SchedulerLogs:
-    def __init__(self, scheduler):
-        logs = Logs(scheduler.get_logs())._repr_html_()
+    def __init__(self, scheduler, start=None):
+        logs = scheduler.get_logs(timestamps=True)
+        if start is not None:
+            logs = [log for log in logs if log[0] > start]
```

Comment:
We could, of course, also add a start/end kwarg to the get_logs and do the filtering in one pass.